data_processors/pipeline/lambdas/demux.py

```diff
@@ -0,0 +1,92 @@
+try:
```

Comment:
This is the workflow dispatcher lambda, right?

Reply:
Yap; It is workflow dispatcher; basically launching workflow per sample after fastq preparation. I guess module `demux` naming here is not appropriate then. Any advice on naming, `dispatcher`?